tracer/build/_build/docker/debian.dockerfile

```diff
@@ -20,7 +20,8 @@ RUN apt-get update \
         ruby \
         ruby-dev \
         rubygems \
-    && gem install --no-document fpm
+    && gem install --no-document fpm \
+    && dotnet dev-certs https
```

Comment:
Not sure why this necessary, when I've never had to do it before, but got an error message when starting a web app saying it was necessary.

Reply:
I don't think this should be necessary, given we have passing builds in CI? 🤷

Reply:
I agree, it's weird, but after a `docker system prune --all`, running:

```
./tracer/build_in_docker.sh BuildTracerHome && \
   ./tracer/build_in_docker.sh RunDotNetSample  -framework net5.0 -SampleName Samples.AspNetCore5

```
fails to start the web server with the attached log:

[RunDotNetSample-failure.txt](https://github.com/DataDog/dd-trace-dotnet/files/7202324/RunDotNetSample-failure.txt)

I'm pretty sure this did used to work without need to call, `dotnet dev-certs https`, but I've no idea what changed.